src/Goerli-DssSpell.sol

```diff
@@ -483,7 +533,7 @@ contract DssSpellAction is DssAction {
         DssExecLib.increaseIlkDebtCeiling(collateral.ilk, collateral.CEIL, /* global = */ true);
 
         // Set stability fee
-        DssExecLib.setIlkStabilityFee(collateral.ilk, collateral.RATE, /* doDrip = */ true);
+        DssExecLib.setIlkStabilityFee(collateral.ilk, collateral.RATE, /* doDrip = */ false);
```

Comment:
why this change? because its new or just not necessary for RWAs or gas savings?

Reply:
Both. Previous RWA onboardings had it set to false.

Reply:
Probably don't need to do the drip for onboardings generally, as there's nothing to accumulate.